From 0505f2b1c0487bf1f2022f034a02228b01e7ddf5 Mon Sep 17 00:00:00 2001
From: Rui Wang <ruwang@redhat.com>
Date: Mon, 21 Jul 2014 03:22:49 -0400
Subject: [redhat] Kconfig: enable earlyprintk-efi on x86

Message-id: <1405912969-15631-5-git-send-email-ruwang@redhat.com>
Patchwork-id: 86049
O-Subject: [PATCH RHEL7.1 BZ1085558 4/4] configs: add earlyprintk-efi config option
Bugzilla: 1085558
RH-Acked-by: Dave Young <dyoung@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Add an entry for CONFIG_EARLY_PRINTK_EFI

Signed-off-by: Rui Wang <ruwang@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/x86_64/CONFIG_EARLY_PRINTK_EFI b/redhat/configs/generic/x86_64/CONFIG_EARLY_PRINTK_EFI
new file mode 100644
index 0000000..b895363
--- /dev/null
+++ b/redhat/configs/generic/x86_64/CONFIG_EARLY_PRINTK_EFI
@@ -0,0 +1 @@
+CONFIG_EARLY_PRINTK_EFI=y
-- 
1.7.1